Documentation/Development.md

```diff
@@ -347,6 +346,39 @@ cp swift-apis/Sources/x10/swift_bindings/xla_tensor_wrapper.h ${DESTDIR}/usr/inc
 ```
 </details>
 
+<details>
+    <summary>What the scripts do.</summary>
```

Comment:
This isn't really what the scripts do.  It is more an explanation of how X10 is developed as X10 cannot be built standalone - it is built as part of TensorFlow, and is current not distributed as part of the TensorFlow repository.  As such we need to explain how to overlay the source structure.

Reply:
I don't see any instructions in this section.  Who do we need to explain this to, and what are they supposed to do with the information?

Reply:
It isn't meant to be instruction, it is covering details of the tree layout to allow a new developer get oriented within the tree to make changes to X10.